k8s/openebs-monitoring/alertmanager.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: alertmanager
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: alertmanager
+  template:
+    metadata:
+      name: alertmanager
+      labels:
+        app: alertmanager
+    spec:
+      serviceAccountName: prometheus
+      containers:
+      - name: alertmanager
+        image: prom/alertmanager:latest
```

Comment:
can we change to the latest release tag. sometime latest versions can become incompatible and this will result in breaking our workflow.

Reply:
Done